common/src/main/java/com/microsoft/identity/common/internal/cache/ADALOAuth2TokenCache.java

```diff
@@ -119,7 +119,7 @@ protected void initializeSharedPreferencesFileManager(final String fileName) {
                 SharedPreferencesFileManager.getSharedPreferences(
                         getContext(),
                         fileName,
-                        -1,
+                        Context.MODE_PRIVATE,
```

Comment:
hmm...isn't it same as per this code here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/10ccbf9fa3236a980500da68c10063153dd68ace/common/src/main/java/com/microsoft/identity/common/internal/cache/SharedPreferencesFileManager.java#L163

Reply:
I believe that's what's being called internally here

Reply:
It should be.  There's no reason not to be explicit, though.  I was worried that I'd done something really strange that was working and shouldn't have been.

Reply:
Given that [`testUpdateTokenCacheUsesResultAuthority` fails in this PR](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/1616) until I remove this line, it is seemingly not the same -- not very obvious what's going on here...

@AdamBJohnsonx as FYI....

Reply:
Ahhh I figured this out... I think what's happening here is that the line above[ where the `key` gets created doesn't account for this behavior](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/10ccbf9fa3236a980500da68c10063153dd68ace/common/src/main/java/com/microsoft/identity/common/internal/cache/SharedPreferencesFileManager.java#L82)... so unless the cache is initialized with the same args everywhere you end up with duplicates "singletons"

Reply:
Realistically, do we just want to remove the option for different modes?

Reply:
@AdamBJohnsonx Probably, but would be take a breaking change to do it?

Reply:
Made this change in an additive way...